recipe/meta.yaml

```diff
@@ -5,32 +5,32 @@
 # confirm that you will not be overwriting a file that has already been uploaded by checking the
 # channel before building.
 
+# Also keep in mind that there must already be a Release or a Tag by the name <version>_<build_number>
+# on the USGS-Astrogeology/ISIS3 repo before invoking the conda build system as this will pull the
+# tar.gz of that name to build. After pushing changes for a release build (so as to
+# include these changes in the release), but before building using the conda build system, be
+# sure to create this Release or Tag.
+
 # This is the version of ISIS that you are building. (Please refer to
 # RFC2 (https://github.com/USGS-Astrogeology/ISIS3/wiki/RFC2:-Release-Process) if you are not sure
-# about what version you are building.)
+# about what version you are building.) 
 # Examples:
 #       A Public Release for ISIS3.6.1:                        {% set version = "3.6.1" %}
 #       A Release Candidate for ISIS3.6.1:                     {% set version = "3.6.1_RC" %}
 #       A custom build of ISIS3.6.1 for the CaSSIS mission:    {% set version = "3.6.1_cassis" %}
 {% set version = "3.7.0_RC" %}
```

Comment:
Should this be `3.7.0_RC2` or does the build number adequately contain the information that it's the second release candidate?

Reply:
Actually, I do think this should change so that we can have a differently-named tag for RC2? (Assuming that github does the naming on the .tar.gz it creates automatically with `{{ version }}_{{ build_number }}.tar.gz` ? ) Or is the assumption that the tag will be named `3.7.0_RC_2`?

Reply:
The latter was the assumption, but we will be going with the former anyhow.